### ISSUE TITLE
unknown command "/go/bin/agent-linux" for "jaeger-agent"

### ISSUE THREAD

**Issue by karanp24** [jaegertracing/jaeger#959]
I am running jaeger agent,query,collector using docker compose and elasticsearch backend. 
Getting error for each component saying the command is unknown. I am using latest images for each of them from docker hub.

Thanks

**Comment by yurishkuro** [comment]
specifying the command is no longer necessary when running Docker images for Jaeger 1.6

which docker-compose files are you using, your own?

**Comment by karanp24** [comment]
No, I followed this blog post, https://cinhtau.net/2018/02/02/jaegertracing-with-elasticsearch/
